components/Layout.vue

```diff
@@ -142,6 +148,8 @@ const nextPage = allLinks[linkIndex + 1];
 const section = navigation.find((s) =>
   s.children.find((child) => child._path === path)
 );
+const { toc } = useContent();
```

Comment:
Here's where I'm using the [document driven composable](https://content.nuxtjs.org/api/composables/use-document-driven)